controller/config.go

```diff
@@ -1268,6 +1297,27 @@ func Validate(c Config) error {
 		}
 	}
 
+	if v, ok := c[SSHServerPort].(int); ok {
+		if v <= 0 {
```

Comment:
How do I disable the ssh server? `0` value could be a great way of turning off the juju ssh server.